modules/administration-guide/partials/proc_verification-custom-registries.adoc

```diff
@@ -0,0 +1,73 @@
+// deploying-the-registries
+
+[id="verification-custom_registries-_{context}"]
+
+.Verification steps
+
+. To verify that the new plug-in is correctly published to the plug-in registry, make a request to the registry path `/v3/plugins/index.json` (or `/devfiles/index.json` for the devfile registry).
++
+[subs="+quotes,+attributes"]
+----
+$ URL=$({orch-cli}  get -o 'custom-columns=URL:.spec.rules[0].host' \
+  -l app=che-plugin-registry {platform-routing} --no-headers)
+$ INDEX_JSON=$(curl -sSL http://$\{URL}/v3/plugins/index.json)
+$ echo $\{INDEX_JSON} | grep -A 4 -B 5 "\"name\":\"my-plug-in\""
+,\{
+ "id": "my-org/my-plug-in/1.0.0",
```

Comment:
This command doesn't work for me:

```
bash-5.0$ echo ${INDEX_JSON} | grep -A 4 -B 5 "\"name\":\"my-plug-in\""
bash-5.0$ ,\{
bash: ,{: command not found
(...)
```

Reply:
Hmm,it seems to work fine for me, except that there is space between field name and value. Maybe it's better to use JQ instead. WDYT?

```bash
~/projects/che-docs runningCustomRegistries > INDEX_JSON=$(curl -sSL http://che-plugin-registry.openshift.io/v3/plugins/index.json)                                17:30:44
~/projects/che-docs runningCustomRegistries > echo ${INDEX_JSON} | grep -A 4 -B 5 "\"name\": \"che-theia\""                                                    INT 17:30:46
    "displayName": "theia-ide",
    "id": "eclipse/che-theia/7.21.1",
    "links": {
      "self": "/v3/plugins/eclipse/che-theia/7.21.1/"
    },
    "name": "che-theia",
    "publisher": "eclipse",
    "type": "Che Editor",
    "version": "7.21.1"
  },
--
    "displayName": "theia-ide",
    "id": "eclipse/che-theia/7.21.2",
    "links": {
      "self": "/v3/plugins/eclipse/che-theia/7.21.2/"
    },
    "name": "che-theia",
    "publisher": "eclipse",
    "type": "Che Editor",
    "version": "7.21.2"
  },
--
```

Reply:
@l0rd Could you test
```
$ INDEX_JSON=$(curl -sSL http://che-plugin-registry.openshift.io/v3/plugins/index.json)                         $ echo ${INDEX_JSON} | yq '.[] | select(.name == "my-plug-in")'                                                     
```
on your end?

Reply:
Also, I've noticed that it may be not easy for user to adapt that command to use for devfile registry.
So, I wonder if it's better to distinct articles for devfile and plugin registries.
With templates mechanism we have, we'll have 0 duplicated code, but users will get ready to copy/paste article )

Reply:
@themr0c WDYT?

Reply:
> @l0rd Could you test
> 
> ```shell
> $ INDEX_JSON=$(curl -sSL http://che-plugin-registry.openshift.io/v3/plugins/index.json)
> $ echo ${INDEX_JSON} | yq '.[] | select(.name == "my-plug-in")'                                                     
> ```
> 
> on your end?

That works

Reply:
did typo. So, changed yq to jq. Because there is two of yq that works fine and the right one is harder to install )